resources/devices/tablet/pegatron.json

```diff
@@ -2,14 +2,14 @@
     "Pegatron Chagall": {
         "standard": false,
         "properties": {
-            "Device_Type": "Mobile Phone",
```

Comment:
This device came up because the Standard file was detecting it as a Tablet with these changes.  Appears to be a tablet:

https://deviceatlas.com/device-data/devices/pegatron/chagall/3876398
https://www.handsetdetection.com/properties/devices/Pegatron/Chagall
http://mobiledevicesinfo.com/pegatron_chagall/